pt_miniscreen/core/components/list.py

```diff
@@ -53,12 +53,31 @@ def __init__(
 
         self._rows_snapshot = None
         self._cleanup_transition = threading.Event()
+        self.visible_scrollbar = True
```

Comment:
I think this should be in state, no reason for someone not to be able to toggle it and it directly relates to the rendered output.

Reply:
good point!